Dockerfile

```diff
@@ -4,7 +4,7 @@ ENV PATH "$PATH:/root/.local/bin/"
 EXPOSE 5000
 RUN mkdir /app
 WORKDIR /app
-COPY poetry.lock poetry.toml pyproject.toml ./
+COPY poetry.lock poetry.toml pyproject.toml .env.test ./
```

Comment:
Copying the `.env.test` might make more sense just in the test stage? It adds another line, but it also allows for slightly less cache-invalidation